misc/bump/bump-ver-manifest

```diff
@@ -33,12 +53,12 @@ then
 fi
 
 echo "Generating new manifest..."
-python3 "$SCRIPT_DIR/ver-bump.py" -m "$MANIFEST_DIR/devel.xml" -r "$latest_hash"
+python3 "$SCRIPT_DIR/ver-bump.py" -m "$MANIFEST" -r "$latest_hash"
 
 echo
 echo "Committing new manifest..."
-git -C "$MANIFEST_DIR" add devel.xml
-git -C "$MANIFEST_DIR" commit -m "Bump kernel revision to '$latest'"
+git -C "$MANIFEST_DIR" add "$MANIFEST_FILE"
+git -C "$MANIFEST_DIR" commit -m "$TAG bump kernel revision to '$latest'"
```

Comment:
You've already included the space in $TAG, which I think is the right thing since you don't want it for non-mcs.
```suggestion
git -C "$MANIFEST_DIR" commit -m "$TAGbump kernel revision to '$latest'"
```

Reply:
Good catch. I think it'll need `${TAG}bump` -- will do that.